### ISSUE TITLE
Shows Status: Failed in Releases. Build Failed. Unable to deploy Balena Sound to fleet. 

### ISSUE THREAD

**Issue by RubixStars** [balena-labs-projects/balena-sound#611]
Shows Status: Failed in Releases. Build Failed. Unable to deploy Balena Sound to fleet. 

1. Click on 1 click deploy button
2. Deploy to a fleet containing raspberry pi 3
3. Go to releases and wait.
4. Shows Status: Failed

Expected for all services to run seamlessly

**Screenshots**
![image](https://user-images.githubusercontent.com/66952734/205354699-b08fb86b-4cce-42a5-8479-975938c3d120.png)
![image](https://user-images.githubusercontent.com/66952734/205354877-0a46e3c6-a01c-4d85-ac54-0a6d42bc29c0.png)
![image](https://user-images.githubusercontent.com/66952734/205354895-398300e0-4c4a-48ae-a786-9b059480e807.png)
![image](https://user-images.githubusercontent.com/66952734/205354924-7de9aefe-b65b-4a2b-9c8f-d6763b003ed2.png)

**Device information**
 - Board Raspberry Pi 3B+
 - Device type Raspberry Pi 3 
 - balenaOS version: balenaOS 2.107.10+rev3

**Comment by maggie0002** [comment]
@andrewn may have an idea?

**Comment by andrewn** [comment]
Is there a reason why the Balena builder wouldn't be able to fetch https://github.com/rust-lang/crates.io-index?

I may be misunderstanding the cargo error though...

```
Step 10/17 : RUN cargo build     --release     --features alsa-backend,pulseaudio-backend     --verbose
 ---> Running in e2531cf3a38f
 
   
Updating
 crates.io index

error
:
 
failed to get `base64` as a dependency of package `librespot v0.4.2 (/app/librespot-src)`


Caused by:

  failed to fetch `https://github.com/rust-lang/crates.io-index`


Caused by:

  unexpected data at the end of the pack; class=Indexer (15)
  ```

**Comment by andrewn** [comment]
Running `balena push gh_andrewn/balena-sound-master-andrewn`

Fails with an out-of-memory error...


```
[spotify]           warning
[spotify]           :
[spotify]            spurious network error (2 tries remaining): failed to mmap. Could not write data: Out of memory; class=Os (2)
[spotify]
```

**Comment by maggie0002** [comment]
@RubixStars, are you using the standard Raspberry Pi 3 image, or the 64bit one? Does it fail on both?

**Comment by andrewn** [comment]
Can you try and build a new release? It should have been fixed in PR #613.

**Comment by saveriogzz** [comment]
I had the same issue and I confirm that the build now ends correctly. Thanks!
